### DISCUSSION TITLE
help with modeling

### DISCUSSION THREAD

**Discussion by brunowilliang** [pocketbase/pocketbase#1311] [Q&A]
I have a collection of auth company and need to register with the companyProfile and also with CompanyWeek? I can register everything in a function?

![Screen Shot 2022-12-20 at 08 48 27](https://user-images.githubusercontent.com/41432581/208660485-96ea9636-4cec-43f0-a9a6-3114ff3e7311.png)

**Comment by ganigeorgiev** [comment]
@Brunowilliang I'm not sure that I understand the questions nor what the collections relation from the screenshot are supposed to do.

If you asking how to create a `company` record with `idCompanyProfile` and `idCompanyWeek`, you have to first create the 2 relations and then use their id in the body data for the `company` create request, eg:

```js
// create relations
const profile = await pb.collection("companyProfile").create({ ... })
const week = await pb.collection("companyWeek").create({ ... })

// create company
const company = await pb.collection("company", {
    "idCompanyProfile": profile.id,
    "idCompanyWeek": week.id,
    // other fields...
})
```
